k-distribution/src/main/scripts/bin/krun

```diff
@@ -70,13 +70,15 @@ execute () {
 
 KX=$(basename "$0")
 
+FoldLines="fold -s"
```

Comment:
Only complaint is that `FoldLines` should likely be `fold_lines` for consistency with bash style